From 5c38d30e71c7a4694b07afa9b230e38da17aef78 Mon Sep 17 00:00:00 2001
From: Songchun Fan <schfan@google.com>
Date: Tue, 1 Dec 2020 12:02:44 -0800
Subject: [PATCH] fix font size scale validator

BUG: 156260178
Test: builds
Change-Id: I32f3b7ece78ec8cc97c52a0484151a6a777aa9da
Merged-In: I32f3b7ece78ec8cc97c52a0484151a6a777aa9da
(cherry picked from commit 3d79b643d4e566d16f7a0338d7d6934f6ca7317e)
---
 .../provider/settings/validators/SystemSettingsValidators.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SettingsProvider/src/android/provider/settings/validators/SystemSettingsValidators.java b/packages/SettingsProvider/src/android/provider/settings/validators/SystemSettingsValidators.java
index bcde58494838..cb610fc61142 100644
--- a/packages/SettingsProvider/src/android/provider/settings/validators/SystemSettingsValidators.java
+++ b/packages/SettingsProvider/src/android/provider/settings/validators/SystemSettingsValidators.java
@@ -89,7 +89,7 @@ public class SystemSettingsValidators {
                         return value == null || value.length() < MAX_LENGTH;
                     }
                 });
-        VALIDATORS.put(System.FONT_SCALE, new InclusiveFloatRangeValidator(0.25f, 5.0f));
+        VALIDATORS.put(System.FONT_SCALE, new InclusiveFloatRangeValidator(0.85f, 1.3f));
         VALIDATORS.put(System.DIM_SCREEN, BOOLEAN_VALIDATOR);
         VALIDATORS.put(
                 System.DISPLAY_COLOR_MODE,
--
2.29.0
